cli/data.go

```diff
@@ -405,27 +410,45 @@ func getMatchingBinaryIDs(ctx context.Context, client datapb.DataServiceClient,
 	}
 }
 
-func downloadBinary(ctx context.Context, client datapb.DataServiceClient, dst string, id *datapb.BinaryID) error {
+func downloadBinary(ctx context.Context, client datapb.DataServiceClient, dst string, id *datapb.BinaryID,
+	httpClient *http.Client, auth authMethod,
+) error {
 	var resp *datapb.BinaryDataByIDsResponse
 	var err error
+	largeFile := false
+	// To begin, we assume the file is large and try getting the binary directly
```

Comment:
[nit] Is this supposed to say we assume the file is small?

Reply:
yep! updated